### ISSUE TITLE
Make bubble stay on screen, bubble customisation

### ISSUE THREAD

**Issue by se-rg-os** [isacan/Andzu#2]
Hey, 

first of all, thank you for awesome library! 👍 

I've just found Dotzu and was wondering if there was something similar for Android and voila!

I have a question regarding bubble:

- is there open API to keep bubble visible on the screen? Right now it is being dismissed after 1-2s
- is there a way to customise bubble somehow? Even by simply using app icon?

**Comment by se-rg-os** [comment]
I've created icon similar to Dotzu icon, I can create PR for it if you wish:

![ic_launcher_round](https://user-images.githubusercontent.com/1237125/28791592-46365a92-762d-11e7-84c6-eafd9aa73180.png)
![ic_launcher](https://user-images.githubusercontent.com/1237125/28791591-463354aa-762d-11e7-93ea-a6e1e7f4bce5.png)

**Comment by isacan** [comment]
Hello Sergii,

Thanks for pr and icon :+1: 

i will add new customization in few days.

Thank you,

**Comment by se-rg-os** [comment]
Here goes PR for icons: https://github.com/isacan/Andzu/pull/3

I've found one issue with unbinding service while testing:
- the app is crashing if to open it first time, use Back button to go from the app, then open it again and use Back again.
- fix is simple and based on Android examples, I can create one more PR for it :)

I still have an issue with bubble disappearing after like 2-5 seconds, and I could not find in the code where exactly it disappears. Do you have any hint?

**Comment by isacan** [comment]
Hello @sergii-frost ,

i fixed disappear problem. can you check it out?

Thanks,
isa can

**Comment by se-rg-os** [comment]
Hey @isacan 
Thank you, I will check it out nearest time, will get back to you after testing.

**Comment by se-rg-os** [comment]
@isacan I tested disappearing issue, it works fine until I open Andzu view.

Then bubble disappears within 1-2 sec and does not reappear if I go back to the Sample app.

----

I have also found one more issue (crash actually), with steps:
1. Start Sample app
2. Open Andzu view
3. Navigate back to Sample app
4. "Close" app by navigating back
5. Open Sample app again
6. "Close" by navigating back again. 

App is crashing with `java.lang.IllegalArgumentException: Service not registered: com.canakkoca.andzu.bubbles.BubblesManager`

I can take a look at this issue, fix it, and create PR.
